tests/lib/UTILS.groovy

```diff
@@ -1,11 +1,398 @@
+        /multiqc_sortmerna\.txt/,
+        /multiqc_sources\.txt/,
+        /multiqc_star\.txt/,
+        /multiqc_star_salmon_deseq2_clustering-plot\.txt/,
+        /multiqc_star_salmon_deseq2_clustering-plot_.*\.txt/,
+        /multiqc_star_salmon_deseq2_pca-plot\.txt/,
+        /picard_deduplication-cnt\.pdf/,
+        /picard_deduplication-cnt\.png/,
+        /picard_deduplication-cnt\.svg/,
+        /picard_deduplication-pct\.pdf/,
+        /picard_deduplication-pct\.png/,
+        /picard_deduplication-pct\.svg/,
+        /picard_deduplication\.txt/,
+        /qualimap_gene_coverage_profile_Counts\.pdf/,
+        /qualimap_gene_coverage_profile_Counts\.svg/,
+        /qualimap_gene_coverage_profile_Normalised\.pdf/,
+        /qualimap_gene_coverage_profile_Normalised\.svg/,
+        /qualimap_genomic_origin-cnt\.pdf/,
+        /qualimap_genomic_origin-cnt\.png/,
+        /qualimap_genomic_origin-cnt\.svg/,
+        /qualimap_genomic_origin-pct\.pdf/,
+        /qualimap_genomic_origin-pct\.png/,
+        /qualimap_genomic_origin-pct\.svg/,
+        /qualimap_genomic_origin\.txt/,
+        /qualimap_rnaseq_genome_results\.txt/,
+        /rsem_assignment_plot\.txt/,
+        /rsem_multimapping_rates\.txt/,
+        /rseqc_bam_stat\.pdf/,
+        /rseqc_bam_stat\.png/,
+        /rseqc_bam_stat\.svg/,
+        /rseqc_bam_stat\.txt/,
+        /rseqc_infer_experiment_plot\.pdf/,
+        /rseqc_infer_experiment_plot\.svg/,
+        /rseqc_inner_distance\.txt/,
+        /rseqc_inner_distance_plot_Counts\.pdf/,
+        /rseqc_inner_distance_plot_Counts\.png/,
+        /rseqc_inner_distance_plot_Counts\.svg/,
+        /rseqc_inner_distance_plot_Counts\.txt/,
+        /rseqc_inner_distance_plot_Percentages\.pdf/,
+        /rseqc_inner_distance_plot_Percentages\.png/,
+        /rseqc_inner_distance_plot_Percentages\.svg/,
+        /rseqc_inner_distance_plot_Percentages\.txt/,
+        /rseqc_junction_annotation_junctions_plot_Events-cnt\.pdf/,
+        /rseqc_junction_annotation_junctions_plot_Events-cnt\.png/,
+        /rseqc_junction_annotation_junctions_plot_Events-cnt\.svg/,
+        /rseqc_junction_annotation_junctions_plot_Events-pct\.pdf/,
+        /rseqc_junction_annotation_junctions_plot_Events-pct\.png/,
+        /rseqc_junction_annotation_junctions_plot_Events-pct\.svg/,
+        /rseqc_junction_annotation_junctions_plot_Events\.txt/,
+        /rseqc_junction_annotation_junctions_plot_Junctions-cnt\.pdf/,
+        /rseqc_junction_annotation_junctions_plot_Junctions-cnt\.png/,
+        /rseqc_junction_annotation_junctions_plot_Junctions-cnt\.svg/,
+        /rseqc_junction_annotation_junctions_plot_Junctions-pct\.pdf/,
+        /rseqc_junction_annotation_junctions_plot_Junctions-pct\.png/,
+        /rseqc_junction_annotation_junctions_plot_Junctions-pct\.svg/,
+        /rseqc_junction_annotation_junctions_plot_Junctions\.txt/,
+        /rseqc_junction_saturation_all\.txt/,
+        /rseqc_junction_saturation_plot_All_Junctions\.pdf/,
```

Comment:
Are there any `/rseqc_junction_saturation_plot/*.*' that DON'T change? If so you can probably collapse these:

```
rseqc_junction_saturation_plot_*_junctions.*
```

Similarly with other file groups. 

?